mono/mini/mini-runtime.c

```diff
@@ -2545,7 +2545,7 @@ mono_llvmonly_runtime_invoke (MonoMethod *method, RuntimeInvokeInfo *info, void
 	gpointer *param_refs;
 	int i, pindex;
 
-	error_init (error);
+	//error_init_internal (error); // FIXME
```

Comment:
Why not just removing it?

Reply:
Still trying to determine which are needed. But agreed, would like to remove (all of them).